enterprise/tiny-comments.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: Tiny Comments
+description: Tiny Comments provides the ability to add comments to the content and collaborate with other users for content editing.
+keywords: enterprise pricing video youtube vimeo mp3 mp4 mov movie clip film comment commenting mediaembed media
```

Comment:
The video-related keywords are not relevant to this page.

Reply:
"collaborate with other users for content editing" doesn't read correctly (to me). 
- Tiny Comments allows content editors to collaborate by providing the ability to add comments to the content.
- Tiny Comments allows users to collaborate by providing the ability to add comments to the content.
- Tiny Comments provides the ability to add comments to content being edited in TinyMCE. This allows TinyMCE users to collaborate without ever leaving the editor.